control_plane/src/bin/neon_local.rs

```diff
@@ -1288,6 +1309,15 @@ async fn try_stop_all(env: &local_env::LocalEnv, immediate: bool) {
 }
 
 fn cli() -> Command {
+    let timeout_arg = Arg::new("timeout-in-seconds")
+        .long("start-timeout")
+        .short('t')
+        .global(true)
+        .help("timeout in seconds until we fail the command")
+        .value_parser(value_parser!(u64))
+        .default_value("10")
+        .required(false);
```

Comment:
Try parsing a Duration here. `humantime::Duration` implements the `FromStr` trait


```suggestion
        .help("timeout in seconds until we fail the command")
        .value_parser(value_parser!(humantime::Duration))
        .default_value("10s")
        .required(false);
```

Reply:
done